compiler/aarch64/codegen/OMRInstOpCodeEnum.hpp

```diff
@@ -477,6 +477,13 @@
 		fmaxd,                                                  	/* 0x1E604800	FMAX      	 */
 		fmins,                                                  	/* 0x1E205800	FMIN      	 */
 		fmind,                                                  	/* 0x1E605800	FMIN      	 */
+	/* Vector Move (2 source) */
+		vorr2d,                                                  	/* 0x4EA01C00   ORR       	 */
+	/* Add (vector) */
+		vadd16b,                                                  	/* 0x4E208400	VADD      	 */
+		vadd8h,                                                  	/* 0x4E608400	VADD      	 */
+		vadd4s,                                                  	/* 0x4E20D400	VADD      	 */
+		vadd2d,                                                  	/* 0x4E60D400	VADD      	 */
```

Comment:
I assume `vadd4s` and `vadd2d` here are for `VectorFloat` and `VectorDouble`, respectively.
Please change the instruction mnemonic to `vfadd4s` and `vfadd2d`, and also change `VADD` in the comment to `FADD`.

Reply:
You are correct, I'll update the opcodes. But for the comments (`VADD`), will it not be better if I change them all to `ADD` as the actual AArch64 instruction, like your previous comment?

Reply:
Sure, I will update the opcodes. But for the comments, will it not be better to use `ADD` for all 4 of the opcodes, as your previous comment? I have checked that the `FADD` instruction is actually different (section- C7.2.49 in ARMv8-A manual). Here, I have used the `ADD` instruction's (C7.2.2) vector variant for different types- `VectorFloat` etc.. Please confirm, thanks.